includes/Integrations/Core_Themes_Support.php

```diff
@@ -93,7 +141,18 @@ public function init() {
 			return;
 		}
 
+		$this->style_handle = 'core-themes/' . sanitize_title( wp_get_theme() );
```

Comment:
```suggestion
		$this->style_handle = 'web-stories-theme-style-' . get_stylesheet();
```

Reply:
Had kept `core-themes/` to build and put core theme style under `css/core-themes/`. else all the core themes will be output into `css/` could become long list.

Other option is using webpack config to output these files separately.

Reply:
Oh, because you are using the `register_style()` method. I see. Hmm.

Then let's output them into `css/`.

Can we name the final files `css/theme-style-(slug).css` or something? Not just `css/twentyten.css` or so.

Reply:
update to use name as suggested in the snippet above `web-stories-theme-styles-{slug}.css`, with the above changes.